graphql_compiler/tests/test_helpers.py

```diff
@@ -26,7 +26,7 @@
 # without requiring a massive number of types and interfaces.
 SCHEMA_TEXT = '''
     schema {
-      query: RootSchemaQuery
+        query: RootSchemaQuery
```

Comment:
The schema text was 2-space indented, which caused editors such as Sublime to try to make the entire file 2-space indented. This is a simple quality-of-life fix, with no functional changes.